Cabal/Cabal.cabal

```diff
@@ -251,11 +257,20 @@ library
     build-depends: directory >= 1.2 && < 1.3,
                    process   >= 1.1.0.2  && < 1.5
 
+  build-tools:
```

Comment:
alex needs to be manually installed in Travis since Cabal is too dumb to install build tools right now.